universal-login-sdk/lib/api/sdk.ts

```diff
@@ -105,16 +110,17 @@ class UniversalLoginSDK {
     this.deploymentObserver = this.deploymentObserver || new DeploymentObserver(this.blockchainService, this.relayerConfig!.contractWhiteList);
   }
 
-  async fetchBalanceObserver(ensName: string) {
-    if (this.balanceObserver) {
-      return this.balanceObserver!;
+  async fetchBalanceObservers(ensName: string) {
+    if (this.balanceObserver && this.aggregateBalanceObserver) {
+      return;
     }
     const walletContractAddress = await this.getWalletContractAddress(ensName);
     ensureNotNull(walletContractAddress, InvalidContract);
     ensureNotNull(this.relayerConfig, MissingConfiguration);
 
     const tokenDetails = await this.getTokensDetails();
     this.balanceObserver = new BalanceObserver(this.balanceChecker, walletContractAddress, tokenDetails);
+    this.aggregateBalanceObserver = new AggregateBalanceObserver(this.balanceObserver!, this.priceOracle);
```

Comment:
There is a possibility that we only use balanceObserver without aggregateBalanceObserver, for example: [subscribeToBalances](https://github.com/UniversalLogin/UniversalLoginSDK/pull/752/files#diff-ddd14f831e8efeac800fe4a489d7fbe0L212)